xlators/cluster/dht/src/dht-linkfile.c

```diff
@@ -165,61 +165,6 @@ dht_linkfile_create(call_frame_t *frame, fop_mknod_cbk_t linkfile_cbk,
     return 0;
 }
 
-int
-dht_linkfile_unlink_cbk(call_frame_t *frame, void *cookie, xlator_t *this,
-                        int32_t op_ret, int32_t op_errno,
-                        struct iatt *preparent, struct iatt *postparent,
-                        dict_t *xdata)
-{
-    dht_local_t *local = NULL;
-    xlator_t *subvol = NULL;
-    char gfid[GF_UUID_BUF_SIZE] = {0};
-
-    local = frame->local;
-    subvol = cookie;
-
-    if (op_ret == -1) {
-        gf_uuid_unparse(local->loc.gfid, gfid);
-        gf_smsg(this->name, GF_LOG_INFO, op_errno, DHT_MSG_UNLINK_FAILED,
-                "path=%s", local->loc.path, "gfid=%s", gfid, "subvolume=%s",
-                subvol->name, NULL);
-    }
-
-    DHT_STACK_DESTROY(frame);
-
-    return 0;
-}
-
-int
-dht_linkfile_unlink(call_frame_t *frame, xlator_t *this, xlator_t *subvol,
```

Comment:
nit: shouldn't it be removed from the .h file as well?

Reply:
Yes